examples/react-native-expo/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  root: true,
+  requireConfigFile: false,
```

Comment:
I always wondered is this actaully a config hahah 🤔

Reply:
I will follow up on this in a subsequent PR - it seems as tho a lot of things with config files whether it's babel, eslint, etc. while supporting many different formats of their config files the IDE only looks to support certain kinds.